.github/workflows/compile_and_release.yml

```diff
@@ -70,21 +70,33 @@ jobs:
         with:
           toolchain: stable
           target: x86_64-unknown-linux-gnu
-        run: cargo build --release --target $TARGET
+          
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --target ${{with.target}}
         
         if: startsWith(matrix.config.os, 'macos') 
         uses: actions-rs/toolchain@v1
         with:
           toolchain: stable
           target: x86_64-apple-darwin # will probably have to change this around to accomodate for M1 & M2
```

Comment:
can you add the aarch64 (silicon chip) build for macOS?

the target is aarch64-apple-darwin

Reply:
resolved